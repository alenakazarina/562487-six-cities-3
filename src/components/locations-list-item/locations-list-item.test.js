import React from 'react';
import renderer from 'react-test-renderer';
import LocationsListItem from './locations-list-item';

const city = `Amsterdam`;
const mockFn = () => {};

describe(`LocationsListItem`, () => {
  it(`should render LocationsListItem`, () => {
    const tree = renderer.create(
        <LocationsListItem
          city={city}
          isActive={true}
          onTabClick={mockFn}
        />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it(`should render LocationsListItem with active class`, () => {
    const tree = renderer.create(
        <LocationsListItem
          city={city}
          isActive={false}
          onTabClick={mockFn}
        />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
