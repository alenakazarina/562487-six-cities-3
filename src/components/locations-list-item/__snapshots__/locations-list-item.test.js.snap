// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LocationsListItem should render LocationsListItem 1`] = `
<li
  className="locations__item"
>
  <a
    className="locations__item-link tabs__item tabs__item--active"
    onClick={[Function]}
  >
    <span>
      Amsterdam
    </span>
  </a>
</li>
`;

exports[`LocationsListItem should render LocationsListItem with active class 1`] = `
<li
  className="locations__item"
>
  <a
    className="locations__item-link tabs__item "
    onClick={[Function]}
  >
    <span>
      Amsterdam
    </span>
  </a>
</li>
`;
